### DISCUSSION TITLE
Possibility to run a script after (and before?) every Export run

### DISCUSSION THREAD

**Discussion by Kraftwurm** [mifi/lossless-cut#1657] [Ideas]
It would be great if I could configure a script that runs after/before every Export run. (Some params could be given , like: filename, number of clips, "before/after" flag, etc. etc.

**Comment by mifi** [comment]
run a script after export, i can understand, but what's the use case of running it before? See #980

**Reply by Kraftwurm** [reply]
I don't really know tbh and you are right, why complicate things if not needed. :)
Me, myself and I would be very happy with just an "AFTER" script feature, that's for sure!
